packages/front-end/components/Experiment/EditVariationsForm.tsx

```diff
@@ -111,14 +114,17 @@ const EditVariationsForm: FC<{
         setVariations={(v) => {
           form.setValue(
             "variations",
-            v.map((data, i) => {
+            v.map((data) => {
+              // eslint-disable-next-line @typescript-eslint/no-explicit-any
+              const newData: any = { ...data };
+              newData.key = data.value;
```

Comment:
question: Why does the key come named as `value` here? Is it possible to change that to be `key` before here so we don't have to do this renaming? Unsure about the level of effort tho

Reply:
it's because the widget is also used for feature rules which do not have a key and instead use value to achieve the same thing.